server/lib/job-queue/job-queue.ts

```diff
@@ -63,12 +63,13 @@ class JobQueue {
     this.jobRedisPrefix = 'bull-' + CONFIG.WEBSERVER.HOST
     const queueOptions = {
       prefix: this.jobRedisPrefix,
-      redis: {
-        host: CONFIG.REDIS.HOSTNAME,
-        port: CONFIG.REDIS.PORT,
-        auth: CONFIG.REDIS.AUTH,
-        db: CONFIG.REDIS.DB
-      }
+      redis: Object.assign({},
+        (CONFIG.REDIS.AUTH && CONFIG.REDIS.AUTH != null) ? { password: CONFIG.REDIS.AUTH } : {},
+	      (CONFIG.REDIS.DB) ? { db: CONFIG.REDIS.DB } : {},
```

Comment:
Is this an extra indent?

Reply:
Ah right! Thanks for spotting it, that didn't show at all in my editor :slightly_frowning_face: